### PR TITLE
Develocity: replace deprecated "GRADLE_ENTERPRISE_ACCESS_KEY" environment variable by "DEVELOCITY_ACCESS_KEY"

### PR DESCRIPTION
**Pull request by murdos**
Following #9596

Fix warning: 
image

### CODE REVIEW

File: .github/workflows/github-actions.yml [hunk 1]
```diff
@@ -115,7 +115,7 @@ jobs:
         run: sed -i '/- gradle-java/d' src/main/resources/config/application.yml
       - name: 'Test: run backend tests'
         env:
-          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
+          DEVELOCITY_ACCESS_KEY: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
```
Comment (murdos) [inline .github/workflows/github-actions.yml:118]: @pascalgrimaud : we should probably also rename the secret from `GRADLE_ENTERPRISE_ACCESS_KEY` to `DEVELOCITY_ACCESS_KEY`

**Review by atomfrede — APPROVED** [review]

**Comment by pascalgrimaud** [comment]
@murdos : just added you as admin of this repository

I don't know the original value of GRADLE_ENTERPRISE_ACCESS_KEY to change it :-)

**Comment by atomfrede** [comment]
I know the original value :)

**Comment by pascalgrimaud** [comment]
@atomfrede : added you as admin

**Pull request merged**